aidial_adapter_openai/app_config.py

```diff
@@ -0,0 +1,39 @@
+from typing import Dict, List
+
+from pydantic import BaseModel, Field
+
+import aidial_adapter_openai.env as env
+
+
+class ApplicationConfig(BaseModel):
+    MODEL_ALIASES: Dict[str, str] = Field(default_factory=dict)
+    DALLE3_DEPLOYMENTS: List[str] = Field(default_factory=list)
+    GPT4_VISION_DEPLOYMENTS: List[str] = Field(default_factory=list)
+    MISTRAL_DEPLOYMENTS: List[str] = Field(default_factory=list)
+    DATABRICKS_DEPLOYMENTS: List[str] = Field(default_factory=list)
+    GPT4O_DEPLOYMENTS: List[str] = Field(default_factory=list)
+    GPT4O_MINI_DEPLOYMENTS: List[str] = Field(default_factory=list)
+    AZURE_AI_VISION_DEPLOYMENTS: List[str] = Field(default_factory=list)
+    API_VERSIONS_MAPPING: Dict[str, str] = Field(default_factory=dict)
+    COMPLETION_DEPLOYMENTS_PROMPT_TEMPLATES: Dict[str, str] = Field(
+        default_factory=dict
+    )
+    DALLE3_AZURE_API_VERSION: str = Field(default="2024-02-01")
+    NON_STREAMING_DEPLOYMENTS: List[str] = Field(default_factory=list)
+
+    @classmethod
+    def from_env(cls) -> "ApplicationConfig":
+        return cls(
+            MODEL_ALIASES=env.MODEL_ALIASES,
```

Comment:
Why not inline definitions from `env` here? since `env` module isn't used anywhere but by this module anyway.